src/T2DMap.cpp

```diff
@@ -4535,6 +4535,29 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     e->ignore();
 }
 
+void T2DMap::changeMapperSettings(int roomSize, int lineSize, bool round, bool info, bool ids, bool viewOnly, bool showPanel)
+{
+  if (mpHost) {
+    this->mBubbleMode = round;
```

Comment:
:information_source: No need to use `this->` on members of this class, also `round` might not be best name to use as it could be mistaken for the **[`std::round`](https://en.cppreference.com/w/cpp/numeric/math/round)** function. I'd suggest `bubbleMode` instead.